tests/acceptance/features/webUIFilesDetails/fileDetails.feature

```diff
@@ -61,14 +71,14 @@ Feature: User can open the details panel for any file or folder
     And the user has browsed to the favorites page
     When the user opens the sidebar for folder "simple-folder" on the webUI
     Then the app-sidebar should be visible
-    And the thumbnail should be visible in the app-sidebar
     And the "details" details panel should be visible
+    And the "big" preview of thumbnail should be visible in the "details" panel
     When the user switches to "actions" panel in details panel using the webUI
     And the "actions" details panel should be visible
+    And the "small" preview of thumbnail should be visible in the "actions" panel
     When the user switches to "people" panel in details panel using the webUI
     Then the "people" details panel should be visible
-    When the user switches to "links" panel in details panel using the webUI
```

Comment:
Why remove this step?

Reply:
@SwikritiT this steps works same as `When the user switches to "people" panel in details panel using the webUI` when observing. so i removed it and instead checked the version panel which was missing.

Reply:
@SagarGi Okay. I think there's still that step in other scenarios too. Can you check if they can be remove them as well?

Reply:
A separate issue can be created for it. i will make a separate issue to check and remove it from scenarios.